framework/requestMocker.js

```diff
@@ -0,0 +1,19 @@
+/*global page*/
+import teremock from "teremock";
+
+const DEFAULT_INTERCEPTOR_CAPTURE = {
+    resourceTypes: "xhr,fetch"
+};
+
+export default class RequestMocker {
+
+    async stop() {
+        await teremock.stop();
+    }
+
+    async start(interceptors = DEFAULT_INTERCEPTOR_CAPTURE) {
+
+        const targetDir = `./mockData/${jasmine["currentSuite"].fullName}/${jasmine["currentTest"].description}`;
+        await teremock.start({ page, wd: targetDir, interceptors: { interceptors } });
+    }
+}
```

Comment:
There is no constructor, class should be refactored to single ton pattern, like i.e.: https://github.com/devbridge/test-juggler/blob/master/framework/interceptor.js